### ISSUE TITLE
changes in runc to pause the restored container

### ISSUE THREAD

**Issue by gitsvm** [opencontainers/runc#2560]
I want to pause the restored container as soon as it restores the container without running for even few ms. 
Not able to get what and  where i make changes in runc code. Please help.

**Comment by kolyshkin** [comment]
@avagin PTAL if this is possible at all.

**Comment by avagin** [comment]
I don't think that we need to play with SIGSTOP here. We can tell CRIU to restore a freezer cgroup into the frozen state.
